contracts/utils/Strings.sol

```diff
@@ -64,4 +65,11 @@ library Strings {
         require(value == 0, "Strings: hex length insufficient");
         return string(buffer);
     }
+
+    /**
+     * @dev Converts an `address` with fixed length of 20 bytes to its not checksummed ASCII `string` hexadecimal representation.
+     */
+    function addressToHexString(address addr) internal pure returns (string memory) {
```

Comment:
Any reason not to call this `toHexString`? It shouldn't conflict with the other function.

Reply:
Before I actually opened this PR, I made the change with function overloading (as you are proposing now) but after thinking through it I decided to deliberately call it `addressToHexString` since all other functions of this library `toString(uint256)`, `toHexString(uint256)`, and `toHexString(uint256,uint256)` assume a `uint256` value type as first parameter. Thus, when I read a function starting with `to` I assumed a `uint256`. I'm not really opinionated about this and if you want I can change it back, but my reasoning was really to make a distinction between `uint256` and `address` input.

Reply:
IMO `toString(address)` would be nice. For library functions like this, I think its reasonable to rely on overloading

Reply:
Fine, fixed :-D